build/ps-entry.sh

```diff
@@ -430,6 +430,12 @@ if [[ $originalArgOne == mongo* ]]; then
 	if [ -f "${MONGO_SSL_DIR}/ca.crt" ]; then
 		CA="${MONGO_SSL_DIR}/ca.crt"
 	fi
+	LDAP_SSL_DIR=${LDAP_SSL_DIR:-/etc/openldap/certs}
+	if [ -f "${LDAP_SSL_DIR}/ca.crt" ]; then
+		tee -a /etc/openldap/ldap.conf <<EOF
+			TLS_CACERT ${LDAP_SSL_DIR}/ca.crt
+EOF
```

Comment:
do we want to add this line and and print or just add this line? if the later let's just do `echo TLS_CACERT ${LDAP_SSL_DIR}/ca.crt >> /etc/openldap/ldap.conf`

if you want to use heredoc anyway, let's use `<<-EOF` and have a proper indentation for `EOF` (don't forget to use Tab instead of spaces for EOF in this case)

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/1480/commits/6cf0de7c3651698e2523c0c6afee5630be94c304